packages/mobile/src/screens/trending-screen/TrendingLineup.tsx

```diff
@@ -0,0 +1,109 @@
+import { useCallback, useEffect, useState } from 'react'
+
+import { useNavigation } from '@react-navigation/native'
+import { Name } from 'audius-client/src/common/models/Analytics'
+import TimeRange from 'audius-client/src/common/models/TimeRange'
+import { makeGetLineupMetadatas } from 'audius-client/src/common/store/lineup/selectors'
+import { setTrendingTimeRange } from 'audius-client/src/common/store/pages/trending/actions'
+import {
+  trendingWeekActions,
+  trendingMonthActions,
+  trendingAllTimeActions
+} from 'audius-client/src/common/store/pages/trending/lineup/actions'
+import {
+  getDiscoverTrendingAllTimeLineup,
+  getDiscoverTrendingMonthLineup,
+  getDiscoverTrendingWeekLineup
+} from 'audius-client/src/common/store/pages/trending/selectors'
+import { isEqual } from 'lodash'
+
+import { Lineup } from 'app/components/lineup'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { make, track } from 'app/utils/analytics'
+
+const getTrendingWeekLineup = makeGetLineupMetadatas(
+  getDiscoverTrendingWeekLineup
+)
+
+const getTrendingMonthLineup = makeGetLineupMetadatas(
+  getDiscoverTrendingMonthLineup
+)
+
+const getTrendingAllTimeLineup = makeGetLineupMetadatas(
+  getDiscoverTrendingAllTimeLineup
+)
+
+const selectorsMap = {
+  [TimeRange.WEEK]: getTrendingWeekLineup,
+  [TimeRange.MONTH]: getTrendingMonthLineup,
+  [TimeRange.ALL_TIME]: getTrendingAllTimeLineup
+}
+
+const actionsMap = {
+  [TimeRange.WEEK]: trendingWeekActions,
+  [TimeRange.MONTH]: trendingMonthActions,
+  [TimeRange.ALL_TIME]: trendingAllTimeActions
+}
+
+type TrendingLineupProps = {
+  timeRange: TimeRange
+  header?: any
```

Comment:
Can this be `header?: SectionListProps<any>['ListHeaderComponent']`

Reply:
yeah had a plan here with `Pick<LineupProps, 'header' | ...` but forgot, added